client.go

```diff
@@ -399,6 +399,10 @@ func (c *Client) OpenFile(path string, f int) (*File, error) {
 }
 
 func (c *Client) open(path string, pflags uint32) (*File, error) {
+	fi, err := c.Stat(path)
```

Comment:
When calling `Create()` this function is called. Except it could be the case that the path does not already exist (we are after all, expecting to create a new file), which means that this c.Stat returns a “file does not exist” error, and so this function now returns nil, “does not exist” rather than actually creating a new file.